docs/user/user_doc.md

```diff
@@ -53,6 +53,111 @@ poll_url = start_response["url"]
+[
+    {
+        "id": "64dce914adc92335dc35beb5",
+        "code": "aix-2c-8g-od",
+        "type": "on-demand",
+        "cores": 2,
+        "memory": 8,
+        "hourlyCost": 0.12
+    },
+    {
+        "id": "64dceafdadc92335dc35beb6",
+        "code": "aix-2c-8g",
+        "type": "always-on",
+        "cores": 2,
+        "memory": 8,
+        "hourlyCost": 0.096
+    },
+    ...
+]
+```
+
+Find a supported function type that best describes your model's purpose. Note down the function's ID.
+```console
+$ aixplain list functions [--api-key <TEAM_API_KEY>]
+{
+    "total": 55,
+    "pageTotal": 55,
+    "items": [
+        {
+            "id": "language-identification",
+            "name": "Language Identification",
+            "output": [
+                {
+                    "name": "Label",
+                    "code": "data",
+                    "dataType": "label",
+                    "defaultValue": []
+                }
+            ],
+            "params": [
+                {
+                    "code": "text",
+                    "name": "Text",
+                    "required": true,
+                    "isFixed": false,
+                    "dataType": "text",
+                    "dataSubType": "text",
+                    "multipleValues": false
+                }
+            ]
+        },
+        ...
+    ]
+}
+```
+
+Once you have chosen a suitable host machine and function, register your model and create an image repository:
+
+```console
+$ aixplain create repo --name <model_name> --hosting-machine <machine_code> --version <model_version> --description <model_description> --function <function_id> --is-async <model_is_asynchronous or not> --source-language <source_language> [--api-key <TEAM_API_KEY>]
+{
+    "repoName": <model_repository_name>,
+    "modelId": <model_id>
+}
+```
+
+This returns a model_id and a repo_name. Next, obtain login credentials for the newly created repository:
+
+```console
+$ aixplain get repo-login [--api-key <TEAM_API_KEY>]
+{
+    "username": <username>,
+    "password": <password>,
+    "registry": <registry_url>
+}
+```
+
+These credentials are valid for 12 hours, after which you much again log in for a fresh set of valid credentials. If you are using Docker, you can use these credentials to log in with the following:
+```console
+docker login --username $USERNAME --password $PASSWORD
+```
+
+Next, tag your image to match the registry and repository name given in the previous steps. If you are using Docker, this would look like the following:
+```console
+$ docker tag <prev_image> {$REGISTRY}/{$REPO_NAME}:<your-choice-of-tag>
+```
+
+Push the newly tagged image to the corresponding repository:
+```console
+$ docker push {$REGISTRY}/{$REPO_NAME}:<your-choice-of-tag>
+```
+
+Once this is done, onboard the model:
+```console
+$ aixplain onboard model --model-id <model_id> --image-tag <model_image_tag> --image-hash <model_image_hash> [--api-key <TEAM_API_KEY>]
```

Comment:
Can you please mention when the image-hash becomes a necessary input? Can the users just use image tag instead?
The image hash should only be a part of the email received by the customer.

What is the response by the CLI on this request apart from getting the email?

Reply:
The image hash is always necessary to ensure that our model onboarding team knows exactly which image we need to onboard. We can't just use the tag because the user may push multiple images with the same tag before asking to onboard, in which case the image we must onboard becomes ambiguous. The customer never receives an email; the onboarding receives the email alert telling us to onboard the model. The user receives no CLI response.